src/__tests__/provider-behavior.test.ts

```diff
@@ -295,31 +364,30 @@ describe('`createDagJWS method`', () => {
     const res = await did.createDagJWS(data)
     const encPayload = await encodePayload(data)
 
+    expect(async () => {
+      await did.verifyJWS(res.jws, {
+        issuer: `did:pkh:eip155:1:${wallet.address}`,
+        capability: cacao,
+        atTime: Date.parse('2021-10-30T16:25:24.000Z'),
```

Comment:
The `atTime` needs to be a unix timestamp still, as. that's what Ceramic will pass: https://github.com/ceramicnetwork/js-ceramic/blob/1b4465167b70292145657fc5e272b7e79effeb2c/packages/common/src/stream.ts#L115

Cacao needs to convert this to an RFC data internally to do the comparison.

Reply:
Date.parse returns a unix timestamp from a string

Reply:
the internal repr in Cacao will be handled in the PR there, shouldnt affect the working of this i think

Reply:
Well, it returns the UNIX timestamp in ms, while Ceramic will pass the UNIX timestamp in seconds.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Reply:
Ah, yes. Should I just divide by 1000?

Reply:
Actually I might be wrong here. Seems like we might actually use ms internally in Ceramic: https://github.com/ceramicnetwork/js-did/blob/main/src/did.ts#L296

Reply:
Investigating...

Reply:
Yep! `valueOf` returns timestamp in ms. So I guess this is fine, and I will change internal workings of CACAO to use the RFC in a PR there.

Reply:
No, I'm trying to figure out what js-ceramic does. The code I linked from js-did might be a bug.

Reply:
https://github.com/ceramicnetwork/js-ceramic/blob/1b4465167b70292145657fc5e272b7e79effeb2c/packages/stream-caip10-link-handler/src/__tests__/caip10-link-handler.test.ts#L55 these tests seem to suggest it is in seconds.

Reply:
https://github.com/ceramicnetwork/js-ceramic/blob/1b4465167b70292145657fc5e272b7e79effeb2c/packages/stream-tile-handler/src/__tests__/tile-document.test.ts#L106 this too, as `commitData.timestamp` is set equal to `proof.blockTimestamp` in a couple of places

Reply:
It makes sense, block timestamp is always in seconds. `js-did` might be buggy after all.